conda/recipes/libcugraph/meta.yaml

```diff
@@ -116,14 +159,24 @@ outputs:
       number: {{ GIT_DESCRIBE_NUMBER }}
       string: cuda{{ cuda_major }}_{{ date_string }}_{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
       ignore_run_exports_from:
+        {% if cuda_major == "11" %}
+        - {{ compiler('cuda11') }}
+        {% else %}
         - {{ compiler('cuda') }}
```

Comment:
```suggestion
```

Reply:
Thanks, pushed to [2886162](https://github.com/rapidsai/cugraph/pull/3456/commits/28861623245f3cb3714d42c0e119f3906d8140bc)